docs/usage/models.md

```diff
@@ -76,55 +81,34 @@ assert user.id == 321
 
 This model is mutable so field values can be changed.
 
-### Model properties
+### Model methods and properties
 
 The example above only shows the tip of the iceberg of what models can do.
 Models possess the following methods and attributes:
 
-`model_dump()`
-: returns a dictionary of the model's fields and values;
-  cf. [exporting models](exporting_models.md#modeldict)
-
-`model_dump_json()`
-: returns a JSON string representation `model_dump()`;
-  cf. [exporting models](exporting_models.md#modeljson)
-
-`copy()`
-: returns a copy (by default, shallow copy) of the model; cf. [exporting models](exporting_models.md#modelcopy)
-
-`model_validate()`
-: a utility for loading any object into a model with error handling if the object is not a dictionary;
-  cf. [helper functions](#helper-functions)
-
-`parse_raw()`
-: a utility for loading strings of numerous formats; cf. [helper functions](#helper-functions)
-
-`parse_file()`
-: like `parse_raw()` but for file paths; cf. [helper functions](#helper-functions)
-
-`from_orm()`
-: loads data into a model from an arbitrary class; cf. [ORM mode](#orm-mode-aka-arbitrary-class-instances)
-
-`model_json_schema()`
-: returns a dictionary representing the model as JSON Schema; cf. [schema](schema.md)
-
-`schema_json()`
-: returns a JSON string representation of `schema()`; cf. [schema](schema.md)
-
-`model_construct()`
-: a class method for creating models without running validation;
-  cf. [Creating models without validation](#creating-models-without-validation)
-
-`model_fields_set`
-: Set of names of fields which were set when the model instance was initialised
-
-`model_fields`
-: a dictionary of the model's fields
-
-`__config__`
-: the configuration class for the model, cf. [model config](model_config.md)
-
-## Recursive Models
+- `copy()`: returns a copy (by default, shallow copy) of the model. See
```

Comment:
```suggestion
- `model_copy()`: returns a copy (by default, shallow copy) of the model. See
```
Not sure if this is right, but there is a `model_copy` method and I assume that's what we want here?

(Also, I do like that you've included them in alphabetical order and would encourage that, I also think we should reorder them in the actual `main.py` file..)